includes/forum-database.php

```diff
@@ -160,6 +162,17 @@ public function buildDatabase() {
             KEY parent_id_id (parent_id, id)
             ) $charset_collate;";
 
+            $sql[] = 'CREATE TABLE '.$this->tables->edit_history." (
+            id int(11) NOT NULL auto_increment,
+            post_id int(11) NOT NULL default '0',
+            user_id int(11) NOT NULL default '0',
+            edited_content longtext NOT NULL,
+            edit_timestamp datetime DEFAULT '0000-00-00 00:00:00' NOT NULL,
```

Comment:
several issues with this one: If you convert this date to unix timestamp, you will get `-62169985172` - 62 billion seconds prior to 1970, and this will only work as long as your database is not running in `sql_mode=TRADITIONAL` - it will fail if the sql_mode is *any* of `SQL_MODE=NO_ZERO_DATE,NO_ZERO_IN_DATE,TRADITIONAL`


- safe alternatives include `DEFAULT '1970-01-01 00:00:00' NOT NULL`   and `DEFAULT NULL NULL`